addons/l10n_in/data/account_tax_template_data.xml

```diff
@@ -65,6 +65,13 @@
         <field name="sequence">6</field>
         <field name="report_id" ref="tax_report"/>
     </record>
+    <record id="tax_report_line_zero_rated" model="account.tax.report.line">
+        <field name="name">Zero Rated</field>
+        <field name="tag_name">Zero Rated</field>
+        <field name="parent_id" ref="tax_report_line_gst_others"/>
+        <field name="country_id" ref="base.in"/>
```

Comment:
There is no country_id field on account.tax.report.line in master :stuck_out_tongue:.

Reply:
The fix is here https://github.com/odoo/odoo/pull/46494 😄